src/main/kotlin/nl/info/zac/smartdocuments/rest/RestSmartDocumentsTemplateGroup.kt

```diff
@@ -49,3 +49,14 @@ private fun convertTemplateGroupResponseToRest(
         groups = group.templateGroups?.map { convertTemplateGroupResponseToRest(it) }?.toSet(),
         templates = group.templates?.map { RestSmartDocumentsTemplate(it.id, it.name) }?.toSet()
     )
+
+fun Set<RestSmartDocumentsTemplateGroup>?.group(groupName: String) =
```

Comment:
Why the nullable check? I mean what's the point of calling this extension function in the first place if your object is null? The null check should be done before calling this function, not in it I would say?

Reply:
Changed the code to throw exception on missing template group